js/components/utils.js

```diff
@@ -529,6 +529,21 @@ var Utils = {
     return (src && (typeof src === 'string' || src instanceof String))
   },
 
+  /**
+   * Returns sanitized config data
+   *
+   * @function sanitizeConfigData
+   * @param {Object} data to be sanitized
+   * @returns {Object} data if data is valid, empty object if not
+   */
+  sanitizeConfigData: function(data) {
+    if (data && (!Array.isArray(data))) {
+      return data
+    } else {
+      return {};
```

Comment:
For our debugging purposes can we get a debug log statement in the negative case. To at least let the developers or our team know that bad data is coming in from somewhere.